run.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+TAG=${TAG:-test}
+
+docker run -t --rm -v ${PWD}:/data asciicast2movie:${TAG} $*
```

Comment:
A helper script to run the built image, assumes the tag is `test` but can be changed with `TAG=1.2.3 ./run.sh`